subworkflows/local/prepare_references.nf

```diff
@@ -84,13 +84,17 @@ workflow PREPARE_REFERENCES {
 
     ARRIBA_DOWNLOAD()
 
+    //cosmic_usr = params.cosmic_usr ?: ""
+    //cosmic_passwd = params.cosmic_passwd ?: ""
+
     AGAT_SPADDINTRONS(
         [[:],gtf],
         []
     )
 
     METAFUSION_GENEBED(
-        AGAT_SPADDINTRONS.out.gff
+        AGAT_SPADDINTRONS.out.gff,
+        params.ensembl_version
```

Comment:
instead of adding a new input channel, can we just add ensembl version to the meta map? you can add it like so:
```
AGAT_SPADDINTRONS(
        [[id:params.ensembl_version],gtf],
        []
    )
```
might be a more nf-core style of doing things. that process will output a channel with the same meta-map, and then metafusion_genebed will have the value.